gui-apps/swappy/Manifest

```diff
@@ -0,0 +1 @@
+DIST swappy-1.2.1.tar.gz 648773 BLAKE2B f60b079d7bcb28708afb8d016f1b997a7b7860b84e614812832e4d6e9bcd2b7f546f544ee2dba6e0e913434f5d13c2377e84419811b3661ffbfb9be1ca61d0be SHA512 e724283845d5555e560129e976dafa1c03c134a3b685b7264138b18b3c7c6a6ea7c4de5862e0b8b5ec3101ec3ba0aeff939e8acee44d672d5eae67517609bb07
```

Comment:
Could you please update to the latest release? Sorry it's taken a while, we prioritize reviewing existing packages before adding new packages to main tree.